### ISSUE TITLE
Bug in Compute temp/chunk

### ISSUE THREAD

**Issue by akohlmey** [lammps/lammps#74]
Posted on lammps-users:

Dear Developers,

I believe there is a bug in the remove_bias() and restore_bias() functions in ComputeTempChunk. This issue will cause a seg fault when using a temp/chunk with keyword "com yes" as the temperature compute in fix nvt. I think it's just an off by one error:

L692 and L729 of compute_temp_chunk.cpp (May 14 2016 version):
```
int index = cchunk->ichunk[i];
should be changed to:
int index = cchunk->ichunk[i]-1;
```
The same correction may also be needed for remove_bias_all() and restore_bias_all() on L714 and L752. Also, I'm a little confused about whether these two functions are correct. As far as I can tell, vbias is never set in temp/chunk. I may be mistaken but I think L716, for example, should be changed to: 
```
v[i][0] -= vcmall[index][0];
```
Thanks,
-David

**Comment by akohlmey** [comment]
Confirmed. These are likely leftovers from the original code template.
addressed in commit 732f2113ed9331ab4f854e20bc16689b68c6ba67

**Comment by akohlmey** [comment]
Feedback from David:

Looks good, thank you Axel.
